File: pbwebwcl/src/app/documenti-progetto/commons/dto/documento.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

export class Documento {
    constructor(
        public tipoDocumento: string,
        public nomeFile: string,
        public numeroProtocollo: string,
        public dataProtocollo: string,
        public dataMarcaturaTemporale: string,
        public dataVerificaFirma: string,
        public dataInserimento: string,
        public idDocumentoIndex: number,
        public idDomanda: number,
        public repository: string,
        public idAllegato: number,
        public fonte: string
    ) { }
}

File: pbwebcert/pbwebcert-jar/src/main/java/it/csi/pbandi/pbwebcert/dto/certificazione/Documento.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbwebcert.dto.certificazione;

public class Documento implements java.io.Serializable {
	private static final long serialVersionUID = 1L;
	private String nomeFile;
	private String tipoDocumento;
	private String idDocumentoIndex;
	
	public void setNomeFile(String val) {
		nomeFile = val;
	}

	public String getNomeFile() {
		return nomeFile;
	}

	public void setTipoDocumento(String val) {
		tipoDocumento = val;
	}

	public String getTipoDocumento() {
		return tipoDocumento;
	}

	public void setIdDocumentoIndex(String val) {
		idDocumentoIndex = val;
	}

	public String getIdDocumentoIndex() {
		return idDocumentoIndex;
	}

	public Documento() {
		super();

	}

	public String toString() {
		return super.toString();
	}
}
